Document/0x08-Testing-Tools.md

```diff
@@ -6,166 +6,163 @@ To perform security testing different tools are available in order to be able to
+ - SUPER: A command-line application that can be used in Windows, macOS and Linux, that analyzes .apk files in search for vulnerabilities - https://github.com/SUPERAndroidAnalyzer/super
+ - SpotBugs: Static analysis tool for Java - https://spotbugs.github.io/
 
 #### Dynamic and Runtime Analysis
 
-- [Android Tcpdump](https://www.androidtcpdump.com "TCPDump") - A command line packet capture utility for Android.
-- [Cydia Substrate: Introspy-Android](https://github.com/iSECPartners/Introspy-Android "Introspy Android") - Blackbox tool to help understand what an Android application is doing at runtime and assist in the identification of potential security issues.
-- [Drozer](https://www.mwrinfosecurity.com/products/drozer/ "Drozer") - Drozer allows you to search for security vulnerabilities in apps and devices by assuming the role of an app and interacting with the Dalvik VM, other apps' IPC endpoints and the underlying OS.
-- [Inspeckage](https://github.com/ac-pm/Inspeckage "Inspeckage") - Inspeckage is a tool developed to offer dynamic analysis of Android apps. By applying hooks to functions of the Android API, Inspeckage will help you understand what an Android application is doing at runtime.
-- [logcat-color](https://github.com/marshall/logcat-color "Logcat color") - A colorful and highly configurable alternative to the adb logcat command from the Android SDK.
-- [VirtualHook](https://github.com/rk700/VirtualHook "VirtualHook") - VirtualHook is a hooking tool for applications on Android ART(>=5.0). It's based on VirtualApp and therefore does not require root permission to inject hooks.
-- [Xposed Framework](https://forum.xda-developers.com/xposed/xposed-installer-versions-changelog-t2714053 "Xposed Framework") - Xposed framework enables you to modify the system or application aspect and behavior at runtime, without modifying any Android application package(APK) or re-flashing.
-- [jdb](https://docs.oracle.com/javase/7/docs/technotes/tools/windows/jdb.html) - The jdb tool is a Java Debugger which allows you to set breakpoints and print application variables. Jdb uses the JDWP protocol.
-- [Andbug](https://github.com/swdunlop/AndBug) - A Scriptable Android Debugger; AndBug is a debugger targeting the Android platform's Dalvik virtual machine intended for reverse engineers and developers. andbug is using the Java Debug Wire Protocol (JDWP
-- [Introspy-Android](https://github.com/iSECPartners/Introspy-Android) - Blackbox tool to help understand what an Android application is doing at runtime and assist in the identification of potential security issues. Limitations: Introspy worked based on 'Cydia Substrate', so to work we need the Cydia and based on the app website Cydia, Supported Android on versions 2.3 through 4.3. On the mentioned version the Introspy does working correctly. (http://www.cydiasubstrate.com/)
```

Comment:
Can you also please add this one again?

Reply:
Please also rewrite it to address the english language errors. Then we can also close this https://github.com/OWASP/owasp-mstg/pull/1106